src/isar/state_machine/states/initialize.py

```diff
@@ -47,11 +51,18 @@ def _run(self) -> None:
 
             try:
                 self.initialize_thread.get_output()
+
             except ThreadedRequestNotFinishedError:
                 time.sleep(self.state_machine.sleep_time)
                 continue
-            except RobotException as e:
-                self.logger.error(f"Initialization of robot failed. Error: {e}")
+
+            except (RobotInitializeException, RobotException) as e:
```

Comment:
Might be enough to catch RobotException here, as RobotInitializeException inherits from RobotException

Reply:
Same argument as above.

Reply:
Fine by me